be/log.h

```diff
@@ -335,6 +335,18 @@ struct m0_be_log_record {
 	/** Pointer to the previous record inside log */
 	m0_bindex_t          lgr_prev_pos;
 	m0_bcount_t          lgr_prev_size;
+	/**
+	 * BE log header flh_discarded pointer during
```

Comment:
pointer updated during?

Reply:
OK

Reply:
The fix is incorrect.

Changed the wording. Please recheck.